packages/core-web/src/index.js

```diff
@@ -50,6 +50,41 @@ function detectAndApplyFixedHeaderStyles() {
         margin-top: calc(-${headerHeight}px - ${bufferHeight}rem);
         height: calc(${headerHeight}px + ${bufferHeight}rem);
       }`);
+  insertCss(`.nav-menu-open { max-height: calc(100% - ${headerHeight}px); }`);
+}
+
+function addResizeHeaderListener() {
+  const headerSelector = jQuery('header[fixed]');
+  const bufferHeight = 1;
+  const resizeObserver = new ResizeObserver(() => {
+    const headerHeight = headerSelector.height();
+    const sheets = document.styleSheets;
+    for (let i = 0; i < sheets.length; i += 1) {
+      const rules = sheets[i].cssRules;
+      // eslint-disable-next-line
```

Comment:
```suggestion
      // eslint-disable-next-line lodash/prefer-get
```

a little more specific

Reply:
Thanks! Updated :)